 <launch>
    <node pkg="socketcan_bridge" type="socketcan_bridge_node" name="can_bridge_node" output="screen">
        <param name="~can_device" value="can0" />
        <remap from="sent_messages" to="can1_tx" />
        <remap from="received_messages" to="can1_rx" />
    </node>

    <node pkg="robstride_actuator_bridge" type="robstride_control_test" name="motor_ctrler" output="screen"/>

 </launch>
